widgets/more_buttons.py

```diff
@@ -35,6 +36,12 @@ def add_buttons(self, option = None):
                 self.add_button(obj, "Add Route", "add_route")
         elif isinstance(obj, (Route, RoutePoint)):
             self.add_button(obj, "Add Route Points", "add_routepoints")
+        elif isinstance(obj, KartStartPoints):
+            #if len(obj.positions) == 0:
+            #^ this check should be performed when/if separate battle mode support is added
+            #or, someone needs to look into multiple starting points for regular courses
+            #i suspect that it's useless to add more than 1 point
```

Comment:
I feel it'd be best not to keep the comment, because it's not even clear whether this check should added.

Reply:
I am not removing the comment because it serves as a reminder for someone to look into this.